packages/suite/src/views/wallet/send/index.tsx

```diff
@@ -28,7 +34,33 @@ interface SendLoadedProps extends UseSendFormProps {
 // separated to call `useSendForm` hook at top level
 // children are only for test purposes, this prop is not available in regular build
 const SendLoaded = ({ children, ...props }: SendLoadedProps) => {
-    const sendContextValues = useSendForm(props);
+    const [isCoinjoinPausedAutomatically, setIsCoinjoinPausedAutomatically] = useState(false);
+
+    const coinjoinAccount = useSelector(state =>
+        selectCoinjoinAccountByKey(state, props.selectedAccount.account.key),
+    );
+
+    const dispatch = useDispatch();
+
+    const sendContextValues = useSendForm({ ...props, coinjoinAccount });
+
+    const accountKey = props.selectedAccount.account.key;
+    const accountType = props.selectedAccount.account?.accountType;
+    const isSessionPaused = coinjoinAccount?.session?.paused;
+
+    // pause coinjoin when send form is open to avoid changing UTXOs while preparing transaction or breaking a coinjoin round
+    // automatically resume coinjoin upon leaving the send form
+    useEffect(() => {
+        if (accountType === 'coinjoin' && !isSessionPaused) {
+            dispatch(pauseCoinjoinSession(accountKey));
```

Comment:
pausing coinjoin session is async, right? so what prevents this to be called multiple times? 

what about this? 
```suggestion
        if (accountType === 'coinjoin' && !isSessionPaused && !isCoinjoinPausedAutomatically) {
            dispatch(pauseCoinjoinSession(accountKey));
```

Or is it really synchronous and the extra condition is not needed?

What if pausing of coinjoin session failed?

Reply:
when I try it, it behaves well...

Reply:
`pauseCoinjoinSession` is indeed synchronous, unlike `restoreCoinjoinSession`. I am adding the suggested extra conditionto be safe in case the implementation changes. ef0b1f6d21d91d0c2e7dad11d6183a0a846547ef